lib/BackgroundJob/RemoveEmptyRooms.php

```diff
@@ -48,22 +50,23 @@ public function __construct(Manager $manager, ILogger $logger) {
 	}
 
 	protected function run($argument) {
```

Comment:
Shouldn't `$this->numDeletedRooms` be set to `0` here? Or is the class instantiated for every iteration?

Reply:
Yeah, this is php. Everything is new all the time ;)